src/domain/event/eventForm/EventForm.tsx

```diff
@@ -482,7 +483,16 @@ const EventForm: React.FC<EventFormProps> = ({
                   </Section>
                 </>
               ) : (
-                <SummarySection isEditingAllowed={isEditingAllowed} />
+                <>
```

Comment:
Hmm... could it be so that the <SummarySection /> wouldn't exist at all if isExternalUser is false... -> { isExternalUser ? <SummarySection .... /> : null} ?

Reply:
My bad... just change the '' return to null?